src/devices/BrickPi3/BrickPi3.csproj

```diff
@@ -13,6 +13,7 @@
   </ItemGroup>
 
   <ItemGroup>
+    <Compile Include="..\..\Iot.Device.Bindings\MathHelper.cs" Link="MathHelper.cs" />
```

Comment:
Usually files like these would go under src/devices/Common instead of IoT.Device.Bindings folder.

Reply:
I moved it there